Dockerfile

```
@@ -23,51 +22,17 @@ USER dev
 WORKDIR /home/dev
 
 # ===================================================================
-FROM base as pyenv
-
-RUN curl https://pyenv.run | bash
-RUN <<EOT bash
-    echo 'export PYENV_ROOT="\$HOME/.pyenv"' >> ~/.bashrc
-    echo 'command -v pyenv >/dev/null || export PATH="\$PYENV_ROOT/bin:\$PATH"' >> ~/.bashrc
-    echo 'eval "\$(pyenv init -)"' >> ~/.bashrc
-    echo 'export PYENV_ROOT="\$HOME/.pyenv"' >> ~/.profile
-    echo 'command -v pyenv >/dev/null || export PATH="\$PYENV_ROOT/bin:\$PATH"' >> ~/.profile
-    echo 'eval "\$(pyenv init -)"' >> ~/.profile
-EOT
-
-# ===================================================================
-FROM pyenv as py38
-RUN pyenv install 3.8
+FROM base as rye
+ARG PYTHON_VERSION=3.11
+RUN curl -sSf https://rye.astral.sh/get | RYE_INSTALL_OPTION="--yes" bash &&\
+    rye pin ${PYTHON_VERSION}
 
 # ===================================================================
-FROM pyenv as py39
-RUN pyenv install 3.9
-
-# ===================================================================
-FROM pyenv as py310
-RUN pyenv install 3.10
-
-# ===================================================================
-FROM pyenv as py311
-RUN pyenv install 3.11
-
-# ===================================================================
-FROM pyenv as shell
-
-
-RUN curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/get-pants.sh | bash
-RUN <<EOT bash
-    echo 'export PATH="$PATH:\$HOME/.local/bin"' >> ~/.bashrc
-EOT
+ENV PATH="${PATH}:/home/dev/bin"
```

In my testing this needs to be:
```suggestion
ENV PATH="${PATH}:/home/dev/.rye/shims"
```